sample/db/samples/product_option_types.rb

```diff
@@ -5,6 +5,14 @@
 size = Spree::OptionType.find_by!(presentation: "Size")
 color = Spree::OptionType.find_by!(presentation: "Color")
 
-solidus_tshirt = Spree::Product.find_by!(name: "Solidus T-Shirt")
-solidus_tshirt.option_types = [size, color]
-solidus_tshirt.save!
+clothes = Spree::Product.where(
+  name: [
+    "Solidus T-Shirt", "Solidus Long Sleeve", "Solidus Women's T-Shirt",
+    "Solidus Snapback Cap", "Solidus Hoodie Zip", "Ruby Hoodie",
+    "Ruby Hoodie Zip", "Ruby Polo"
+  ]
```

Comment:
Do all of these products have size/color variants? If yes 👍, if no what's the benefit of adding the option types when there's only the master variant?

Reply:
So I'll try to express again the thought process behind these changes:

1. I installed a fresh solidus app locally
2. Noticed that Solidus Women's T-Shirt did not have any variants (size or colour)
3. Went to admin panel to add the variants manually
4. Couldn't find how to add a variant

After further investigation I understood that it was because the option types were not assigned to the product. 

I just thought it would be a good idea to skip the step of adding the option type and be able to add the variant from the get go.

Reply:
The thing is that you add one or more option types to a product when that product has variants. Having an option type assigned but no variants might be considered an inconsistency because it means we are assuming that the product has some variants, but we didn't create them yet. 

On a side note, in terms of consistency, with the clothing products we are creating in the demo, it's acceptable not to have the color option type, but maybe the size option type and its variants (eg. s/m/l) should always be present.

My proposal is to add the size option types and variants to all of them and keep the `Solidus T-Shirt` with both. WDYT?

Reply:
ok, will modify as suggested

Reply:
By the way, maybe it's not clear or easy to add a variant to a product without option types. If you have any suggestions on how to improve it, let me know!

Reply:
Fixed in https://github.com/solidusio/solidus/pull/4680/commits/5875a59b43db3c69e4508fe03f297070b9511170

Reply:
> My proposal is to add the size option types and variants to all of them and keep the Solidus T-Shirt with both. WDYT?

If that's what we want to communicate, I think it'd be better to refactor instead of:

1. Manually select clothing products and add size & color option types.
2. Manually select the rest of the clothing products and add the size option type.

Do:

1. Manually select clothing products and add color option type.
2. Select all clothing products and add the size option type.

That way, we don't need to visually confirm that we do not forget any clothing product to add the size, plus we're safe if we add new items.

WDYT?

Reply:
Implemented you suggestion in https://github.com/solidusio/solidus/pull/4680/commits/9eda486ef6821034f6957802c7473ff672aba51a